src/main/java/org/folio/persist/AbstractRepository.java

```diff
@@ -72,4 +74,11 @@ protected AbstractRepository(PostgresClient postgresClient, String tableName,
   public Future<RowSet<Row>> update(SQLConnection connection, String id, T record) {
     return update(succeededFuture(connection), id, record);
   }
+
+  public Future<RowSet<Row>> deleteAll() {
+    final String removeAllQuery = format("DELETE FROM %s_mod_inventory_storage.%s",
+      tenantId, tableName);
+
+    return postgresClientFuturized.execute(removeAllQuery);
+  }
```

Comment:
The implementation of this method seems a bit odd (and risky?). Can't `postgresClient` do that?

Reply:
Changed to `postgresClient.delete(tableName, new Criterion())`